distutils/command/install_data.py

```diff
@@ -56,6 +57,16 @@ def run(self):
                     )
                 (out, _) = self.copy_file(f, self.install_dir)
                 self.outfiles.append(out)
+            elif isinstance(f, Path):
```

Comment:
This covers the case that one of the data files is a simple pathlib.Path, but it doesn't cover the case if a `pathlib.Path` is used in a tuple (in the else block below).

Reply:
Ah you are right. Should I fix it or do you want to fix it?

Reply:
I'm on it. But thanks for offering.

Reply:
My thinking is that `convert_path` already wraps all the places where a Path might be encountered, so probably that should be expanded to account for `pathlib.Path`.